### PR TITLE
operators / it-academy / JC1-117-22 / Victor Polonnikov / Feature solution tasks01 06

### PR DESCRIPTION
**Pull request by VictPol**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [x] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java [hunk 1]
```diff
@@ -30,6 +29,11 @@ public static void main(String[] args) {
     static String getPlayingTime(int playingTimeInSeconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int D = playingTimeInSeconds / 60 / 60 / 24;
+        int H = playingTimeInSeconds % (60 * 60 * 24) / 60 / 60;
+        int m = playingTimeInSeconds % (60 * 60) / 60;
+        int s = playingTimeInSeconds % 60;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java:35]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java [hunk 2]
```diff
@@ -36,6 +35,7 @@ public static void main(String[] args) {
     static int calculateDeviationStandardWeight(int height, int weight) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return weight - (height - 100);
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java:38]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java [hunk 3]
```diff
@@ -35,6 +34,7 @@ public static void main(String[] args) {
     static float getDistance(float milliseconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
+        return milliseconds / 1000 * SOUND_SPEED * 1000 / 3600;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:38]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java [hunk 4]
```diff
@@ -31,6 +30,7 @@ public static void main(String[] args) {
     static int getNumberFullTrucks(int cargo, int carryingCapacity) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return cargo / carryingCapacity;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java:34]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java [hunk 5]
```diff
@@ -32,6 +31,7 @@ public static void main(String[] args) {
     static int getAmountRemainingCargo(int cargo, int carryingCapacity) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        return cargo % carryingCapacity;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java:35]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java [hunk 6]
```diff
@@ -35,6 +34,7 @@ public static void main(String[] args) {
     static float getDistance(float milliseconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:37]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java [hunk 7]
```diff
@@ -36,6 +35,7 @@ public static void main(String[] args) {
...
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java:39]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java [hunk 8]
```diff
@@ -30,6 +29,11 @@ public static void main(String[] args) {
...
+        return D + " " + H + ":" + m + ":" + s;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java:37]: "Violation: Useless blank lines
Solution: Remove blank lines"

**Review by rakovets — APPROVED** [review]

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java (hunk 3 above)
Comment (VictPol) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:38]: Like this?
static final double GRAVITATIONAL_CONSTANT = 9.81;

double potentialEnergy(double mass, double height) {
  return mass * height * GRAVITATIONAL_CONSTANT;
}

**Pull request closed without merging**